### ISSUE TITLE
Why use Dropout in the evaluation

### ISSUE THREAD

**Issue by JuliRao** [NVIDIA/tacotron2#364]
https://github.com/NVIDIA/tacotron2/blob/0102db28e2e167146b20ab72a9e8dc17d78f3a4f/model.py#L99

Why is this dropout always set to true, even in evaluation ??

I try to set replace this line as dropout=self.training, and the synthesis result is much worse, which is quite wield.

**Comment by Dekakhrone** [comment]
Duplicate of #247
